katas/6kyu/counting-duplicates.js

```diff
@@ -0,0 +1,17 @@
+function duplicateCount(text) {
+    const letters = text.toLowerCase().split('');
+    let counter = 0;
+    let memo = {};
+
+    if (!letters.length) return 0;
+
+    letters.forEach(letter => {
```

Comment:
`forEach` is more about modifying items inside an existing array. in your case you just use items to check something without change the exact item, so it'd be better to use `map`.
The reason why I'm asking about it, because it'd be better to have a habit to use always `map` and change it to `forEach` only when it's really necessary.
The same practice that I recommend everyone for `const` and `let`. use `const` everytime and change it to `let` only it's necessary

Reply:
Thank you! Do I need to replace **forEach** method with **map** here?
So I use **let** when the value of variable will be changed. Both variables here will be changed.